otel-api-common/test-suite/Test/OTel/API/Common/LoggingSpec.hs

```diff
@@ -255,7 +255,7 @@ expectedQueueElems shouldIgnoreLoc queue expectedElems = do
       -- convenient to temporarily change the @expectationFailure@ to the
       -- following:
       --
-      actualElems `shouldBe` expectedElems'
+      -- actualElems `shouldBe` expectedElems'
```

Comment:
Do you want to leave this?

Reply:
I had meant to comment it out in #2 but forgot. There's a comment above shedding a little light on it, but it's not too descriptive. The assertion message below the commented `shouldBe` is more useful in the general case, as it indicates a timeout when waiting for the STM check to match. It's nice to see that message if there's an error, but it doesn't give the nice diffing like `shouldBe` gives. So if things are changed in the future and a test starts failing, we know first that it's from an STM check timing out. While working through understanding the test failure, we can uncomment the `shouldBe` to temporarily get that nicer diffing.

There is some sneakiness that can be done to attach messages to `shouldBe` errors, via catching the underlying HUnit exception and modifying it. That'd be a better way, as `shouldBe` could always stay here and a message could be tacked onto it indicating the timeout happened. That didn't feel super worth it for now though.